src/linear_algebra/GLA.jl

```diff
@@ -11,6 +11,9 @@ using AutoHashEquals
 using LinearMaps
 import LinearMaps: adjoint
 const LMs = LinearMaps
+using LinearOperators
+import LinearOperators: adjoint
+const LOs = LinearOperators
```

Comment:
I think that we can add the opEye to the import line above and then eliminate this module alias.

Reply:
Looks like this is resolved now, except for the lingering alias `LOs`.

Reply:
Just removed the lingering alias